doc/user/layouts/partials/create-source/key-constraint/syntax.html

```diff
@@ -0,0 +1 @@
+_key_constraint_ | Declare a set of columns as a primary key. For more information, see [`Key constraint details`](#key-constraint-details).
```

Comment:
Not sure, but this could also be **PRIMARY KEY (col_list) NOT ENFORCED**.

Reply:
👍🏼

Reply:
Done in ba74f91c1.